packages/shared/src/components/sidebar/SidebarBottomSection.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import { act, render, RenderResult, screen } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import React from 'react';
+import nock from 'nock';
+import Post from '../../../__tests__/fixture/post';
+import { AuthContextProvider } from '../../contexts/AuthContext';
+import { AlertContextProvider } from '../../contexts/AlertContext';
+import { Alerts } from '../../graphql/alerts';
+import { SidebarBottomSectionSection } from './SidebarBottomSection';
+import { mockGraphQL } from '../../../__tests__/helpers/graphql';
+import { FEED_SETTINGS_QUERY } from '../../graphql/feedSettings';
+
+describe('SidebarBottomSection component', () => {
+  const noop = jest.fn();
+  const defaultPost = Post;
+  const client = new QueryClient();
+  const updateAlerts = jest.fn();
+  const defaultAlerts: Alerts = {
+    lastChangelog: new Date(defaultPost.createdAt).toISOString(),
+  };
+
+  beforeEach(async () => {
+    nock.cleanAll();
+    jest.clearAllMocks();
+    client.clear();
+
+    Object.defineProperty(global, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation((query) => ({
+        matches: query.includes('1020'),
+        addListener: jest.fn(),
+        removeListener: jest.fn(),
+      })),
+    });
+
+    mockGraphQL({
+      request: { query: FEED_SETTINGS_QUERY, variables: { loggedIn: false } },
+      result: { data: { feedSettings: { blockedTags: ['javascript'] } } },
+    });
+  });
+
+  const renderComponent = (): RenderResult => {
+    return render(
+      <QueryClientProvider client={client}>
+        <AuthContextProvider
+          user={null}
+          squads={[]}
+          getRedirectUri={noop}
+          updateUser={noop}
+          tokenRefreshed={false}
+        >
+          <AlertContextProvider
+            alerts={defaultAlerts}
+            updateAlerts={updateAlerts}
+            loadedAlerts
+          >
+            <SidebarBottomSectionSection
+              optOutWeeklyGoal
+              showSettings={false}
+              sidebarExpanded={false}
+              sidebarRendered
+              activePage="/"
+              shouldShowLabel
+            />
+          </AlertContextProvider>
+        </AuthContextProvider>
+      </QueryClientProvider>,
+    );
+  };
+
+  it('should render changelog and badge if available', async () => {
+    client.setQueryData(
+      ['changelog', 'latest-post', { loggedIn: false }],
+      defaultPost,
+    );
+    const lastChangelog = new Date(defaultAlerts.lastChangelog);
+    lastChangelog.setMonth(lastChangelog.getMonth() - 1);
+    defaultAlerts.lastChangelog = lastChangelog.toISOString();
+
+    let changelog: HTMLElement;
+
+    await act(async () => {
```

Comment:
Just general remark, preferably it's best to avoid act, there are some other solution as to test changing elements, was it giving issues in this context?

Reply:
Just a leftover from copy pasting and initial playing with tests setup.. I will remove `act` from places where it's not needed.